### DISCUSSION TITLE
Cannot load gnutls module (

### DISCUSSION THREAD

**Discussion by rolldice0** [inspircd/inspircd#2073] [Support]
**Description**

I have set inspircd up and its fully working plain text. but i want tls support. so i generated a tls cert with certbot and copied the key and cert to my install and gave same ownership/group as the rest of the files, installed pkg-make and libgnutls28-dev (I am on debian 12). I then enabled the gnutls module. This all worked and the module shows as enabled. But in my inspircd.conf, when i try to load the module when i attempt to start inspircd, it says 
"Loading core modules ....................
[*] Loading module:     m_ssl_gnutls.so
[*] Module file could not be found: m_ssl_gnutls.so"

**Steps to reproduce the issue:**

Build inspircd from source on debian 12, install necessary pkgs for gnutls, enable gnutls with ./config, then once enabled try loading the module with your inspircd.conf file

**Describe the results you received:**
The gnu tls module does not load despite being enabled and having its dependencies installed.

**Describe the results you expected:**
the module to load

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `./bin/inspircd --version`:**
3.16.1

**Comment by rolldice0** [comment]
Here is what I have in my inspircd.conf pertaining to gnutls. If I should have anything else, I don't.

```
<bind address="" port="6697" type="clients" ssl="gnutls">
<gnutls
    certfile="/home/user/inspircd-3.16.1/run/conf/cert.pem"
    keyfile="/home/user/inspircd-3.16.1/run/conf/key.pem"
    priority="SECURE192:-VERS-SSL3.0">
<module name="m_ssl_gnutls.so">
```
I also did make install again and it didnt fix it

**Comment by sadiepowell** [comment]
Can you paste the output of the following commands run from the root of your source tree (presumably `/home/user/inspircd-3.16.1`):

```sh
file src/modules/m_ssl_gnutls.cpp

file build/*/modules/m_ssl_gnutls.so 

file run/modules/m_ssl_gnutls.so 
```
